modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a route through an Ingress object
+
+Some ecosystem components have an integration with Ingress resources but not with
+route resources. To cover this case, {product-title} automatically creates
+managed route objects when an Ingress object is created. These route objects are deleted
+when the corresponding Ingress objects are deleted.
+
+.Procedure
+
+. Define an Ingress object in the {product-title} console or by entering the oc `create` command:
++
+.YAML Definition of an Ingress
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: frontend
+  annotations:
+    route.openshift.io/termination: "reencrypt" <1>
+spec:
+  rules:
+  - host: www.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: frontend
+          servicePort: 443
+  tls:
+  - hosts:
+    - www.example.com
+    secretName: example-com-tls-certificate
+----
++
+<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route`
+as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All
```

Comment:
IMO,  we  should also include one more line similar or close to, "Unlike route object, "reencrypt" ingress termination type requires service serving certificates for the backend instead of `"caCertificate"` and `"destinationCACertificate"` certs"

Reply:
You mean _like_ route objects?

Reply:
What I meant is, in a simpler way, *Unlike* `route` objects, ingress objects with "reencrypt" termination type will work only with service serving certificates for the backend SSL connections.

Reply:
Thats not true. If the backend has a TLS cert that is trusted by something in the ingress-controllers global cert store, its not required.

Reply:
Thanks, @quarterpin! Looks like we're all set to merge. If so, I'll go ahead and merge first thing in the morning.